### PR TITLE
Implement `validate` cmd to validate yaml files

### LINKED ISSUE #35: feat: cmd - lint - validate generated yamls
Before we deploy or prior to deploy we want to make sure that our inputted yamls follow our rules. Hence we need to create a cmd that will lint the yamls. We should also have it prior to a install. 

`armada validate -f <path>`

### PR DESCRIPTION
**Pull request by ghost**
**What is the purpose of this pull request?**: validate yaml files from the command line

**What issue does this pull request address?**: Fixes #35 

**Notes for reviewers to consider**: needs to be worked into api next, probably in a different PR?

**Specific reviewers for pull request**:
@gardlt @drewwalters96

### CODE REVIEW

File: armada/handlers/armada.py [hunk 1]
```diff
@@ -82,11 +82,11 @@ def pre_flight_checks(self):
             if ct_type == 'git' and not git.check_available_repo(location):
                 raise ValueError(str("Invalid Url Path: " + location))
 
-            if not self.tiller.tiller_status():
```
Comment (theyer) [inline armada/handlers/armada.py]: @gardlt only tangentially related to this pr but these commands were being run for each chart when they really just need to be run once.

Comment (gardlt) [inline armada/handlers/armada.py]: these checks are only running once before we start running any charts. we do not run them per chart.

Comment (theyer) [inline armada/handlers/armada.py]: the pre_flight_checks method iterates through each chart and these are currently inside the loop

Comment (gardlt) [inline armada/handlers/armada.py]: this looks good.

**Review by gardlt — APPROVED** [review]
LGTM.

**Review by drewwalters96 — APPROVED** [review]
LGTM.

**Pull request merged**